engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -1,26 +1,34 @@
 package cromwell.webservice
 
+import java.sql.Timestamp
+
 import akka.actor.ActorSystem
 import akka.testkit._
-import cromwell.core.WorkflowId
+import cromwell.core.{KnowsWhatTimeItIs, WorkflowId}
 import cromwell.services.MetadataServiceActor._
+import cromwell.services._
 import cromwell.webservice.PerRequest.RequestComplete
-import org.joda.time.DateTime
 import org.scalatest.prop.TableDrivenPropertyChecks
 import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
 import spray.http.{StatusCode, StatusCodes}
 import spray.json._
+
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
-class MetadataBuilderActorSpec extends TestKit(ActorSystem("Metadata")) with FlatSpecLike with Matchers with TableDrivenPropertyChecks with ImplicitSender with BeforeAndAfterAll {
+class MetadataBuilderActorSpec extends TestKit(ActorSystem("Metadata"))
+  with FlatSpecLike with Matchers with TableDrivenPropertyChecks with ImplicitSender with BeforeAndAfterAll {
 
   behavior of "MetadataParser"
 
   val defaultTimeout = 100 millis
   val mockServiceRegistry = TestProbe()
   val parentProbe = TestProbe()
 
+  // TestKit has its own `now` that prevents extending KnowsWhatTimeItIs on this class directly.
+  val knowsWhatTimeItIs = new Object with KnowsWhatTimeItIs
```

Comment:
Yeah I need to do something with this, mixing into a class that already has TestKit is not going to be a weird edge case.